monGARS/api/web_api.py

```
@@ -1,3 +1,48 @@
+
+
+@app.get("/ready")
+async def ready() -> dict:
+    return {"status": "ready"}
+
+
+@app.get("/api/v1/conversation/history", response_model=List[MemoryItem])
+async def conversation_history(
+    user_id: str,
+    limit: int = 10,
+    current_user: dict = Depends(get_current_user),
```

Ruff (0.12.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)